lib/tasks/auto_annotate_models.rake

```diff
@@ -1,7 +1,7 @@
 # NOTE: only doing this in development as some production environments (Heroku)
 # NOTE: are sensitive to local FS writes, and besides -- it's just not proper
 # NOTE: to have a dev-mode tool do its thing in production.
-if Rails.env.development?
```

Comment:
I don't think we need this change now that annotate_models is working again

Reply:
Quite right... I've removed that again.